submissions/Ant-C-tech/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,466 @@
+      href="img/favicon/apple-icon-76x76.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="114x114"
+      href="img/favicon/apple-icon-114x114.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="120x120"
+      href="img/favicon/apple-icon-120x120.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="144x144"
+      href="img/favicon/apple-icon-144x144.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="152x152"
+      href="img/favicon/apple-icon-152x152.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="img/favicon/apple-icon-180x180.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="192x192"
+      href="img/favicon/android-icon-192x192.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="img/favicon/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="96x96"
+      href="img/favicon/favicon-96x96.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="img/favicon/favicon-16x16.png"
+    />
+    <link rel="manifest" href="img/favicon/manifest.json" />
+    <meta name="msapplication-TileColor" content="#ffffff" />
+    <meta
+      name="msapplication-TileImage"
+      content="img/favicon/ms-icon-144x144.png"
+    />
+    <meta name="theme-color" content="#ffffff" />
+    <!-- /Favicon -->
+
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;400;700&family=Rubik&display=swap"
+      rel="stylesheet"
+    />
+    <!-- /Fonts -->
+
+    <link rel="stylesheet" href="styles/normalize.css" />
+    <link rel="stylesheet" href="styles/app.css" />
+
+    <title>Cinema Application</title>
+  </head>
+
+  <body>
+    <video preload="auto" autoplay muted loop poster="img/video_poster.jpg" class="trailer">
+      <source
+        src="video/trailer.mp4"
+        type="video/mp4"
+      />
+    </video>
+
+    <main>
+      <section class="posterSection">
+        <div class="textContent">
+          <h2 class="textContent__title">Vivarium</h2>
+          <p class="textContent__description">
+            On their search for the perfect home, Gemma (Imogen Poots) and Tom
+            (Jesse Eisenberg) visit a new house in a labyrinthine suburban
+            neighbourhood. When they attempt to leave, each road mysteriously
+            takes them back to where they started, leading them on a
+            mind-bending trip, trapped in a surreal nightmare.
+          </p>
+          <a href="#" class="btn-readMore">&#9660;</a>
+        </div>
+
+        <div class="badge badge-rating">
+            <p>IMBd<br>5.8</p>
+        </div>
+
+        <div class="badge badge-data">
+            <p>From<br>01 March<br>1h 39m</p>
+        </div>
+
+        <div class="mask"></div>
```

Comment:
Mabey better use pseudoelement for `.posterSection` instead of creating an empty element.